razor-compiler.sln

```diff
@@ -37,21 +37,21 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Microsoft.CodeAnalysis.Razo
 EndProject
 Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "perf", "perf", "{59F4C41D-F903-47A5-8E78-A908BBC0F532}"
 EndProject
-Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Microsoft.AspNetCore.Razor.Microbenchmarks", "src\Compiler\perf\Microbenchmarks\Microsoft.AspNetCore.Razor.Microbenchmarks.csproj", "{61AF10E6-1746-444C-8CB7-C76F7AA413E8}"
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Microsoft.AspNetCore.Razor.Microbenchmarks.Compiler", "src\Compiler\perf\Microbenchmarks\Microsoft.AspNetCore.Razor.Microbenchmarks.Compiler.csproj", "{61AF10E6-1746-444C-8CB7-C76F7AA413E8}"
```

Comment:
Why do we need to rename these projects?

Reply:
As in the PR description:

> There are duplicate projects in both tooling and compiler. The compiler ones have been renamed to .Compiler to avoid restore clashes. These will get unified at a later date.

I didn't know this, but nuget restore + deps stuff works on just the project name. So if you have two projects with the same name, but different directories, they race when restoring, overwriting each other and causing one to fail (in a non-predictable fashion, too! Because its a race, running restore multiple times gives you different build errors. Fun!)

Reply:
Ah, I missed that. Thanks.